deploy/list-connectors.ts

```diff
@@ -0,0 +1,78 @@
+import { glob } from "tinyglobby";
+import { join } from "node:path";
+import Module from 'node:module';
+import fs from 'fs-extra'
```

Comment:
I think your project is not applied the prettier because semicolon is not written in import statements.
In other statements, somewhere use double quotes and other use single quotes.
I'll fix it